timetracker/entries/views.py

```diff
@@ -1,6 +1,20 @@
-from django.shortcuts import render
+from django.shortcuts import render, get_object_or_404
 
-from .models import Entry
+from .models import Client, Entry, Project
```

Comment:
I prefer:

```
from . import models
```

Especially in light of that bug we looked at last week, it makes it  explicit where the classes are coming from.